mlrun/api/crud/client_spec.py

```diff
@@ -43,6 +46,9 @@ def get_client_spec(self):
             auto_mount_params=self._get_config_value_if_not_default(
                 "storage.auto_mount_params"
             ),
+            default_function_priority_class=self._get_config_value_if_not_default(
+                "default_function_priority_class"
+            ),
```

Comment:
Same for this one, it does not have a default value, but don't send it if it's not set